### ISSUE TITLE
does lightgbm support the normal pairwise ranking without delta ndcg in the gradient?

### ISSUE THREAD

**Issue by chloe-wang** [microsoft/LightGBM#2877]
## Summary

Xgboost has rank:pairwise objective for normal ranking while lightgbm has only lambdarank with delta ndcg in the gradient. But in some cases, we want the overall ranking rather than the top ones.

**Comment by guolinke** [comment]
pair-wise and list-wise are equivalent. list-wise merges the gradient from all pairs to the all elements. 
You can get the overall ranking in list-wise too.

**Comment by chloe-wang** [comment]
thanks! @guolinke
